components/progressbar/index.css

```diff
@@ -10,51 +10,86 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-progressbar.css";
-@import "../vars/css/components/spectrum-meter.css";
-@import "../vars/css/components/spectrum-fieldlabel.css";
+.spectrum-ProgressBar {
+  /* Static tokens */
+  --spectrum-progressbar-animation-ease-in-out-indeterminate: cubic-bezier(.45, 0, .40, 1);
+  --spectrum-progressbar-animation-duration-indeterminate: 1000ms;
+  --spectrum-progressbar-corner-radius: var(--spectrum-corner-radius-100);
+
+  --spectrum-progressbar-fill-size-indeterminate: 70%;
+  /* --spectrum-global-dimension-static-size-2400 */
+  --spectrum-progressbar-size-2400: 192px;
+  /* --spectrum-global-dimension-static-size-2500 */
+  --spectrum-progressbar-size-2500: 200px;
```

Comment:
Maybe we could start keeping track of those missing core-tokens in this Paperdoc:

https://paper.dropbox.com/doc/Missing-tokens-from-Core-Tokens-CSS-Migration--Bm2u3nX_Kd76hNmOVGz71wHkAg-y9o9A3tt2qKFBSzUjfhsY

Reply:
@pfulton I added some of those that I think are missing from my recent work as well.